airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorMetadataDefinitionV0.py

```diff
@@ -160,6 +160,9 @@ class Config:
 
 
 class Data(BaseModel):
+    class Config:
+        extra = Extra.forbid
```

Comment:
The lack of good "heaven forbid" gifs is upsettings

Reply:
Forbid in general is lacking

<img src="https://media3.giphy.com/media/2zYk0N9ilSztOAPzb0/giphy.gif"/>